### ISSUE TITLE
SeedClient not returning any results

### ISSUE THREAD

**Issue by devoid** [ModelSEED/Model-SEED-core#44]
```
lib/ModelSEED/ModelSEEDClients/t/MSSeedSupportClient.t .. 2/? 
#   Failed test 'Expected sequence not found!'
#   at lib/ModelSEED/ModelSEEDClients/t/MSSeedSupportClient.t line 26.
lib/ModelSEED/ModelSEEDClients/t/MSSeedSupportClient.t .. 3/? 500 Error detected by service. at /Users/devoid/Desktop/Core/lib/myRAST/ClientThing.pm line 724
ClientThing::_send_request('MSSeedSupportClient=HASH(0x7ff273030bf8)', 'function', 'getRastGenomeData', 'args', '---\x{a}genome: 315750.3\x{a}password: reviewer\x{a}username: reviewer\x{a}', 'source', 'ClientThing', 'dbName', undef, ...) called at /Users/devoid/Desktop/Core/lib/myRAST/ClientThing.pm line 476
ClientThing::_call_method('MSSeedSupportClient=HASH(0x7ff273030bf8)', 'getRastGenomeData', 'HASH(0x7ff27657ca58)') called at /Users/devoid/Desktop/Core/lib/myRAST/ClientThing.pm line 248
ClientThing::AUTOLOAD('MSSeedSupportClient=HASH(0x7ff273030bf8)', 'HASH(0x7ff27657ca58)') called at lib/ModelSEED/ModelSEEDClients/t/MSSeedSupportClient.t line 32
# Looks like you failed 1 test of 3.
# Looks like your test exited with 255 just after 3.
```

**Comment by devoid** [comment]
I get the following error after calling `blast_sequence()` with the example arguments:

```
""Unable to open File /vol/model-dev/MODEL_DEV_DB/ReactionDB/blastdb/83333.1/JFB4FSw.out: ""
at     /home/chenry/Model-SEED-core/lib/ModelSEED/FIGMODEL.pm line 136
ModelSEED::FIGMODEL::FIGMODELERROR('Unable to open File /vol/model-dev/MODEL_DEV_DB/ReactionDB/bl...')
called at /home/chenry/Model-SEED-core/lib/ModelSEED/FIGMODEL/FIGMODELdatabase.pm line 539
ModelSEED::FIGMODEL::FIGMODELdatabase::load_multiple_column_file(
'ModelSEED::FIGMODEL::FIGMODELdatabase=HASH(0x102adcb8)',
'/vol/model-dev/MODEL_DEV_DB/ReactionDB/blastdb/83333.1/JFB4FS...', '\x{9}') called at 
/home/chenry/public_html/FIGdisk/dist/releases/current/cee/lib/FigKernelPackages/MSSeedSupport.pm line 248
MSSeedSupport::blast_sequence('MSSeedSupport=HASH(0x102b42f0)', 'HASH(0x10666500)') called at /home/chenry/public_html/FIGdisk/dist/releases/current/cee/lib/FigKernelPackages/ServerThing.pm line 1776
eval {...} called at /home/chenry/public_html/FIGdisk/dist/releases/current/cee/lib/FigKernelPackages/ServerThing.pm line 1776
ServerThing::RunRequest('CGI=HASH(0x102adaf0)', 'MSSeedSupport=HASH(0x102b42f0)') called at /home/chenry/public_html/FIGdisk/dist/releases/current/cee/lib/FigKernelPackages/ServerThing.pm line 122
ServerThing::RunServer('MSSeedSupport', undef) called at MSSeedSupport_server.cgi line 78
```

**Comment by devoid** [comment]
This was fixed with commit 368d55d482375bc5f036f1ca4483b0b48660a618.
